app/src/main/java/mozilla/lockbox/view/ItemListFragment.kt

```diff
@@ -129,10 +131,25 @@ class ItemListFragment : CommonFragment(), ItemListView {
         }
     }
 
-    override fun setDisplayName(text: String) {
+    // sets large text in header as either email or display name
+    override fun setDisplayEmailName(text: String) {
+        view!!.navView.menuHeader.displayName.text = text
+    }
+
+    // sets smaller text in header as email
+    override fun setAccountName(text: String) {
         view!!.navView.menuHeader.accountName.text = text
     }
 
+    override fun setAvatarFromURL(url: String) {
+        Picasso.get()
+            .load(url)
+            .placeholder(R.drawable.ic_avatar_placeholder)
+            .resize(80, 80)
+            .transform(CropCircleTransformation())
```

Comment:
1. could we put the `80` someplace more accessible to designers, e.g. `res/attrs.xml`
2. `CropCircleTransformation` do we need a new one of these each time, or should we try and use only one. This is a minor point.

Reply:
1. Done!
2. Not sure, I will take a look at this